src/window/replay_confirm.h

```diff
@@ -0,0 +1,6 @@
+#ifndef WINDOW_CONFIRM_REPLAY_H
+#define WINDOW_CONFIRM_REPLAY_H
+
+void window_confirm_replay_show(void(*close_callback)(void));
```

Comment:
Naming: file is called `replay_confirm` while the constant and functions are called `confirm_replay`